File: Bonaliz.Web/src/lib/tenantConfig.js
let tenantConfig = null;

export function setTenantConfig(config) {
  tenantConfig = {
    nome: config.nome,
    logoUrl: config.logoUrl.trim(), // remover o \r\n se vier
    corPrimaria: config.corPrimaria || "#000000",
    corSecundaria: config.corSecundaria || "#ffffff",
  };
}

export function getTenantConfig() {
  return tenantConfig;
}

File: BonaLiz.Negocio/ViewModels/TenantConfig.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BonaLiz.Negocio.ViewModels
{
    public class TenantConfig
    {
        public string Nome { get; set; }
        public string CorPrimaria { get; set; }
        public string CorSecundaria { get; set; }
        public string LogoUrl { get; set; }
    }
}
